app/brave_main_delegate.cc

```diff
@@ -20,6 +20,7 @@
 #include "brave/components/brave_ads/browser/buildflags/buildflags.h"
 #include "brave/renderer/brave_content_renderer_client.h"
 #include "brave/utility/brave_content_utility_client.h"
+#include "chrome/browser/ui/webui/settings/settings_ui.cc"
```

Comment:
Why do we include this instead of `components/password_manager/core/common/password_manager_features.h`?
Also, please check if we have deps in place for password_manager_features.h and add into deps if not.

Reply:
thanks for pointing this out @yrliou, a typo for sure. Fixed! Looking at the deps right now.